R/join_key.R

```diff
@@ -2,30 +2,35 @@
 #'
 #' @description `r lifecycle::badge("stable")`
 #'
-#' @details `join_key()` will create a relationship for the variables on a pair
-#' of datasets.
+#' @description `Create a relationship for the variables on a pair of datasets.
+#' It is directed by default with `dataset_1` being the parent.
 #'
-#' @param dataset_1,dataset_2 (`character(1)`) dataset names. If `dataset_2` is omitted,
-#'  a primary key for `dataset_1` is created.
-#' @param keys (optionally named `character`) where `names(keys)` are columns in `dataset_1`
-#' corresponding to columns of `dataset_2` given by the elements of `keys`.
+#' @param dataset_1,dataset_2 (`character(1)`) Dataset names. When `dataset_2` is omitted,
+#' a primary key for `dataset_1` is created.
+#' @param keys (optionally named `character`) Column mapping between the datasets,
+#' where `names(keys)` maps columns in `dataset_1` corresponding to columns of
+#' `dataset_2` given by the elements of `keys`.
 #'
 #' If unnamed, the same column names are used for both datasets.
 #'
 #' If any element of the `keys` vector is empty with a non-empty name, then the name is
 #' used for both datasets.
+#' @param parent (`character(1)`) Indicates the parent dataset in a parent-child
+#' relationship or `none` if it is an undirected relationship.
+#' One of `dataset_1`, `dataset_2` or `none`.
```

Comment:
I think it could be more precise (even though I don't like the implementation)

```suggestion
#' @param parent (`character(1)`) Indicates which argument hold a name of a parent dataset in a parent-child
#' relationship or `none` if it is an undirected relationship.
#' One of `"dataset_1"`, `"dataset_2"` or `"none"`.
```